Task_1/Task_1/src/components/Input_form/form.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from "react";
+
+const Form = (props) => {
+  const [enterTask, setEnterTask] = useState("");
+  const [enterDate, setEnterDate] = useState("");
+
+  function TaskHandler(event) {
+    setEnterTask(event.target.value);
+  }
+  function DateHandler(event) {
+    let date = event.target.value.toString();
+    setEnterDate(date);
+  }
+
+  function add_task() {
+    console.log("button is worling");
+    console.log(enterTask);
+    props.add_task(enterTask,enterDate);
+  }
+
+  return (
+    <div className="mt-100px w-full text-center mt-[100px]">
```

Comment:
mt-100px ? will it work @codal-uchoudhary ? At last again mt-[100px] ?

Reply:
Updated